crates/astria-bridge-withdrawer/local.env.example

```diff
@@ -31,10 +31,26 @@ ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_COMETBFT_ENDPOINT="http://127.0.0.1:26657"
 # Chain ID of the sequencer chain which transactions are submitted to.
 ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_CHAIN_ID="astria"
 
+# Set to true to enable frost threshold signing.
+ASTRIA_BRIDGE_WITHDRAWER_FROST_THRESHOLD_SIGNING_ENABLED=false
+
 # The path to the file storing the private key for the sequencer account used for signing
 # transactions. The file should contain a hex-encoded Ed25519 secret key.
+# Only used if `frost_threshold_signing_enabled` is false.
 ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_KEY_PATH=/path/to/priv_sequencer_key.json
 
+# The minimum number of frost participants required to sign a transaction.
+# Only used if `frost_threshold_signing_enabled` is true.
+ASTRIA_BRIDGE_WITHDRAWER_FROST_MIN_SIGNERS=0
+
+# The path to the json-encoded frost public key package.
+# Only used if `frost_threshold_signing_enabled` is true.
+ASTRIA_BRIDGE_WITHDRAWER_FROST_PUBLIC_KEY_PACKAGE_PATH=""
+
+# The frost participant gRPC endpoints.
+# Only used if `frost_threshold_signing_enabled` is true.
+ASTRIA_BRIDGE_WITHDRAWER_FROST_PARTICIPANT_ENDPOINTS=[]
```

Comment:
Avoid using `[]` in env vars because they interact with shell tests (started with `[`).

Maybe a comma-separated can be used to parse the individual URLs out? E.g. `https://url.one,https://url.two` - this relies on `,` being a reserved keyword for the domain part of URLs.